src/test/java/teammates/storage/api/NotificationsDbTest.java

```diff
@@ -0,0 +1,272 @@
+package teammates.storage.api;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.NotificationTargetUser;
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.test.BaseTestCaseWithLocalDatabaseAccess;
+
+/**
+ * SUT: {@link NotificationsDb}.
+ */
+public class NotificationsDbTest extends BaseTestCaseWithLocalDatabaseAccess {
+
+    private final NotificationsDb notificationsDb = NotificationsDb.inst();
+    private final Map<String, NotificationAttributes> typicalNotifications = getTypicalDataBundle().notifications;
+
+    // Notification attributes will be assigned with value at setup()
+    private NotificationAttributes n;
+
+    @BeforeMethod
+    public void setup() throws Exception {
+        for (NotificationAttributes n : typicalNotifications.values()) {
+            notificationsDb.createEntity(n);
+        }
+        n = typicalNotifications.get("notification1");
+    }
+
+    /**
+     * Removes all notifications created by each test.
+     */
+    @AfterMethod
+    public void cleanUp() {
+        List<NotificationAttributes> retrieved = notificationsDb.getAllNotifications();
+        assertNotNull(retrieved);
```

Comment:
Can directly use `typicalNotifications` from above instead of fetching from DB.

Do we need the `assertNotNull(retrieved);` here?